content/blog/getting-started-with-k8s-part2/index.md

```diff
@@ -0,0 +1,380 @@
+---
+title: "Getting Started With Kubernetes: Application Basics"
+date: 2020-06-30
+meta_desc: "A tutorial for getting started with Kubernetes using Infrastructure as Code."
+meta_image: getting-started.png
+authors:
+    - sophia-parafina
+tags:
+    - Kubernetes
+
+---
+
+Welcome to the second article in a series using infrastructure as code to deploy applications with Kubernetes. The series walks you through building a Kubernetes cluster on cloud providers, deploying applications, and “Day 2” activities such as migrating Node groups. In the [previous article]({{< relref "/blog/getting-started-with-k8s-part1" >}}), we showed how to create a Kubernetes cluster for AWS, Azure, and GCP. In this installment, we’ll learn how to deploy an application using Kubernetes objects.
+
+<!--more-->
+
+Kubernetes is complex, but it’s important to keep in mind that Kubernetes is not merely a container orchestrator. It is designed to keep applications highly available and to scale on demand. It accomplishes this by running multiple instances of application containers and replacing them as needed.
+
+There are many moving parts to Kubernetes, so let’s review the objects used to deploy applications in Kubernetes.
+
+## Pods
+
+A Pod is where applications run, and it is the smallest unit of execution for Kubernetes. A Pod hosts one or more containers that make up a complete application. For example, a Pod can have a container running an entire LAMP stack for a web application, or it can run a container with NGINX that connects to a database running in another Pod.
+
+Each Pod has an IP address and can address other Pods in the same network. In the previous article, we deployed clusters in a VPC. Pods deployed in a Kubernetes cluster can communicate with each other via TCP/IP. Containers within a Pod share, e.g., think of containers as a process, share the same IP address and port space, and can communicate via localhost or IPC, such as POSIX shared memory or SystemV semaphores.
+
+Pods have a specific [lifecycle](https://kubernetes.io/docs/concepts/workloads/Pods/Pod-lifecycle/). They are created, assigned a UID, and scheduled to run on a Node, and run until they either fail or are terminated. When a Node dies, Pods are deleted and replaced. Pods have five distinct states or phases:
+
+- *Pending*  The Pod has been scheduled for creation, but one or more containers have not been created.
+- *Running*  The Pod has been created in the Node, all containers have been created, and at least one container is running.
+- *Succeeded* All the containers have been successfully terminated and will not be restarted.
+- *Failed* All containers have been terminated, but at least one container has exited with a failed non-zero status.
+- *Unknown* The state of the Pod was not returned, possibly due to a communication error.
+
+When deploying an application, you can see the status of the Pods.
+
+![Deployment](kubectl-Pod-status.gif)
```

Comment:
GIF seems like it's missing in the preview

Reply:
tyoe